gitbook/reference/migration.md

```
@@ -46,3 +47,11 @@ After migrating, verify the output:
 ```
 
 This checks `.gsd/` integrity and flags any structural issues.
+
+Use `/gsd inspect` for database diagnostics. If a project has markdown artifacts but a missing or damaged database, start GSD once so the database opens, then run:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)